### PR TITLE
update ru translation

### PR DESCRIPTION
**Pull request by X-NicON**
fix intent text.

### CODE REVIEW

**Comment by X-NicON** [comment]
The application for Android, is in demand in Russia/CIS? or it is used by 2.5 people.
( I want to understand whether it is needed to whom this translation )

**Comment by gubatron** [comment]
This translation is intended for everybody that uses that app of course.
You're not supposed to share files you don't own the rights to or share illegal files with this app.

**Comment by gubatron** [comment]
unlicensed != unauthorized.

Not sure if that change was correct.

You can have the creator's written permission, aka an authorization, the file still might be formally unlicensed.

**Comment by X-NicON** [comment]
I read the original string. I realized what was happening.

File: res/values-ru/strings.xml [hunk 1]
```diff
@@ -348,7 +347,7 @@
     <string name="seeding_settings">Настройки раздачи</string>
     <string name="seeding_settings_explanation">Пользователи BitTorrent в сети смогут скачивать части файла, который раздает ваш торрент.</string>
     <string name="seeding_wifi_settings_explanation">Раздавайте только при Wi-Fi подключении. Избегайте дополнительных расходов у своего сотового оператора.</string>
-    <string name="intent_text">FrostWire - это пиринговая программа для обмена только разрешенными файлами. Установка и использование программы не дает право на получение или распространение несанкционированного контента.</string>
+    <string name="intent_text">FrostWire - это пиринговая программа для обмена законно распространяемыми файлами. Установка и использование программы не дает право на получение или несанкционированное распространение контента.</string>
```
Comment (gubatron) [inline res/values-ru/strings.xml]: can you make it here back to "only authorized" files. Not "legally distributed"

File: res/values-ru/strings.xml [hunk 2]
```diff
@@ -449,7 +443,7 @@
     <string name="donate_in_btc">Пожертвовать, биткоинами</string>
     <string name="torrent_transfer_enqueued_on_mobile_data">Торрент начнет загрузку автоматически, при подключении к сети Wi-Fi. (Проверьте настройки, касающиеся использования 3G/LTE)</string>
     <string name="torrent_transfer_consuming_mobile_data">Внимание: Эта торрент загрузка использует 3G/LTE сеть. (В настройках, можно настроить использование BitTorrent только через Wi-Fi)</string>
-    <string name="add_transfer_label">Вставьте ссылку на magnet или торрент</string>
+    <string name="add_transfer_label">Вставьте magnet или торрент ссылку</string>
```
Comment (gubatron) [inline res/values-ru/strings.xml]: If this is easier to understand :+1:

Comment (X-NicON) [inline res/values-ru/strings.xml]: better than it was, but still not perfect

File: res/values-ru/strings.xml (hunk 1 above)
Comment (X-NicON) [inline res/values-ru/strings.xml]: ok, I won't touch these authorized/unauthorized. Apparently I'm just not friends with them in Russian language.
Thanks for the persistence and sorry to waste your time.

**Comment by gubatron** [comment]
Thank you so much!

**Pull request merged**